### PR TITLE
Text Utilities Component

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my text utilities components

this page contains the text utilities components

live link: https://css-companion.netlify.app/components/text-utilities.html

Following are the features of this page

- nav: contains brand name and links to the home page and components library.
- main: contains the documentation of text-utilities components with snippets and examples.
- aside: contains links to all my component libraries.
- footer: contains links to my social media

image

image

image

### CODE REVIEW

File: component/text/text-utilities.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./text.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+    <script src="https://kit.fontawesome.com/e45a62e93a.js" crossorigin="anonymous"></script>
+</head>
+    <title>Document</title>
```
Comment (sahiba0915) [inline component/text/text-utilities.html]: give a proper title

**Review by sahiba0915 — COMMENTED** [review]
good to go

File: component/text/text-utilities.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./text.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
```
Comment (utkarshducoder) [inline component/text/text-utilities.html:9]: remove extra space

**Review by sahiba0915 — APPROVED** [review]

File: component/text/text-utilities.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                            <p class="lower-text">loWERcasE tExt</p>
+                            <p class="upper-text">UppeERcase text</p>
+
+                            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=vscode&wt=bw&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14.5px&lh=155%25&si=false&es=2x&wm=false&code=%2520%2520%2520%253Cp%2520class%253D%2522lower-text%2522%253EloWERcasE%2520tExt%253C%252Fp%253E"
+  style="width: 402px; height: 85px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+
```
Comment (utkarshducoder) [inline component/text/text-utilities.html:109]: here also

**Review by utkarshducoder — APPROVED** [review]

File: component/text/text-utilities.html (hunk 2 above)
Comment (i-am-a-codegirl) [reply component/text/text-utilities.html:9]: okay

**Pull request merged**